core/trino-main/src/test/java/io/trino/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -369,6 +394,18 @@ public void setup()
         }
     }
 
+    @State(Thread)
+    public static class RowTypeBenchmarkData
+            extends TypeBenchmarkData
+    {
+        @Setup
+        public void setup()
+        {
+            RowType type = RowType.anonymous(ImmutableList.of(VARCHAR, BIGINT, LONG_DECIMAL_TYPE));
```

Comment:
You can just encode/decode single bigint field within `RowBlock`. That would be even better because it would expose `RowBlock` overhead even more